### PR TITLE
Add C-Chain RPC API and multicoin support

### PR DESCRIPTION
**Pull request by cgcardona**
C-Chain RPC APIs

* `export`
* `exportAVAX`
* `exportKey`
* `import`
* `importAVAX`
* `importKey`

X-Chain RPC APIs

* `export`
* `import`

And associated tests.

### CODE REVIEW

File: src/apis/evm/api.ts [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+  };
+
+  /**
+   * Send multicoin assets including AVAX from an account on the X-Chain to an address on the C-Chain. This transaction
+   * must be signed with the key of the account that the asset is sent from and which pays
+   * the transaction fee.
+   *
+   * @param username The Keystore user that controls the account specified in `to`
+   * @param password The password of the Keystore user
+   * @param to The ID of the account the asset is sent to.
```
Comment (dasconnor) [inline src/apis/evm/api.ts:82]: ID or address? Same below in importAvax

**Review by dasconnor — COMMENTED** [review]
Couple of nit questions on the documentation, otherwise LGTM

File: src/index.ts [hunk 2]
```diff
@@ -77,27 +83,42 @@ export default class Avalanche extends AvalancheCore {
      * @param protocol The protocol string to use before a "://" in a request,
      * ex: "http", "https", "git", "ws", etc ...
      * @param networkid Sets the NetworkID of the class. Default [[DefaultNetworkID]]
-     * @param avmChainID Sets the blockchainID for the AVM. Will try to auto-detect,
+     * @param XChainID Sets the blockchainID for the AVM. Will try to auto-detect,
```
[the remark this answers is not in the document]
Comment (cgcardona) [reply src/index.ts:86]: The virtual machine is called `AVM` and the instance of it on our default subnet is the `X-Chain` so i think this makes sense.

File: src/apis/evm/api.ts (hunk 1 above)
Comment (cgcardona) [reply src/apis/evm/api.ts]: Good catch. Fixed

**Pull request merged**